react/SlideToggle/SlideToggle.less

```diff
@@ -110,12 +115,12 @@
   opacity: 0;
   transition: 0.3s;
   position: relative;
-  top: 5px;
+  top: -4px;
```

Comment:
This looks off to me, i would actually suggest removing the relative positioning and apply flex centering in the `icon` class above.

Reply:
Hrmmm I'll give it ago, be right back 🚪